### PR TITLE
Add gui

### PR DESCRIPTION
**Pull request by ejscott**
**The Problem or Enhancement**

This addresses issue: # 6

**Explanation of Solution** - Simple GUI added that accesses the scripts then executes when user pushes "Backup" or "Restore". Also added more detailed CONTRIBUTE.md file.

**Testing & Validation Done**
Validated on Mac system. Currently not compatible on Windows. Must have tkinter package.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+You can use `test.sh` to test the encryption and decryption processes.
+
+## Submitting Issues
+
+We use GitHub issues to track public bugs and feature requests. Please ensure your description is clear and has sufficient instructions to be able to reproduce the issue.
+
+## License
+
+By contributing, you agree that your contributions will be licensed under its MIT License.
```
Comment (aarons) [inline CONTRIBUTING.md:40]:
```suggestion
## License

This project is currently not licensed for use. Cloning the repo for personal use on a local system, or for inspiration should be fine. However, integrating into other software systems or using it commercially is not presently licensed. 
```

**Review by aarons — CHANGES_REQUESTED** [review]
Awesome!

One minor request on the contributing markdown file, then I can approve!

**Comment by ejscott** [comment]
Thank you for the feedback! Made adjustments to per request.

**Review by aarons — APPROVED** [review]
Nice!

**Pull request merged**